src/Vegetation/PlantHydraulics.jl

```diff
@@ -72,18 +77,16 @@ struct PlantHydraulicsParameters{FT <: AbstractFloat}
     area_index::NamedTuple{(:root, :stem, :leaf), Tuple{FT, FT, FT}}
     "water conductivity in the above-ground plant compartments (m/s) when pressure is zero, a maximum"
     K_sat::NamedTuple{(:root, :stem, :leaf), Tuple{FT, FT, FT}}
-    "van Genuchten parameter (1/m)"
-    vg_α::FT
-    "van Genuchten parameter (unitless)"
-    vg_n::FT
-    "van Genuchten parameter (unitless)"
-    vg_m::FT
     "porosity (m3/m3)"
     ν::FT
     "storativity (m3/m3)"
     S_s::FT
     "Root distribution function P(z)"
     root_distribution::Function
+    "Conductivity model parameters"
+    retention_params::PS
```

Comment:
I believe these will need to have different symbols for their types, because they wont be the same type.

Reply:
is Ksat now stored inside the conductivity model parameters?

Does that mean each compartment has the same Ksat? I think that is OK because we are talking about conductivity, but if we used conductance this would change. is that correct?

(our previous code let Ksat be different for each)

Reply:
Hey Katherine, just removed K_sat from above and yes thinking we should start with this as easier. Eventually K_sat_root should be different though because its magnitude is much smaller than that of the stem and leaves.

Reply:
If we load plant conductance in the future instead of plant conductivity, I think it is correct that the conductance would be different for each compartment because conductivity is the inherent plant property whereas conductance depends on length

We could still load only one plant conductance (=plant conductivity/length of plant from tip of root to top of leaf) in the parameter struct though and multiply by compartment length inside the functions which require units of m/s; or we could pass plant conductance*length compartment directly.